analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DisposableNotDisposed.CSharp9.cs

```diff
@@ -95,3 +97,18 @@ public void Foo()
         var s = new WebClient();                                   // Noncompliant - another tracked type
     }
 }
+
```

Comment:
Pedantic: we usually add also a link to the related issue if there is one open: https://github.com/SonarSource/sonar-dotnet/issues/5879